data/actions/scripts/other/music.lua

```diff
@@ -1,5 +1,73 @@
+local instruments = {
+	[2070] = {effect = CONST_ME_SOUND_GREEN}, -- wooden flute
+	[2071] = {effect = CONST_ME_SOUND_GREEN}, -- lyre
+	[2072] = {effect = CONST_ME_SOUND_GREEN}, -- lute
+	[2073] = {effect = CONST_ME_SOUND_GREEN}, -- drum
+	[2074] = {effect = CONST_ME_SOUND_GREEN}, -- panpipes
+	[2075] = {effect = CONST_ME_SOUND_GREEN}, -- simple fanfare
+	[2076] = {effect = CONST_ME_SOUND_GREEN}, -- fanfare
+	[2077] = {effect = CONST_ME_SOUND_GREEN}, -- royal fanfare (actual effect is unknown)
+	[2078] = {effect = CONST_ME_SOUND_GREEN}, -- post horn
+	[2079] = {effect = CONST_ME_SOUND_GREEN}, -- war horn
+	[2080] = {effect = CONST_ME_SOUND_BLUE}, -- piano (actual effect is unknown)
+	[2081] = {effect = CONST_ME_SOUND_BLUE}, -- piano
+	[2082] = {effect = CONST_ME_SOUND_BLUE}, -- piano
+	[2083] = {effect = CONST_ME_SOUND_BLUE}, -- piano
+	[2084] = {effect = CONST_ME_SOUND_BLUE}, -- harp (actual effect is unknown)
+	[2085] = {effect = CONST_ME_SOUND_BLUE}, -- harp
+	[2332] = {effect = CONST_ME_SOUND_BLUE}, -- Waldo's post horn (actual effect is unknown)
+	[2367] = {effect = CONST_ME_SOUND_GREEN}, -- drum
+	[2368] = {effect = CONST_ME_SOUND_GREEN}, -- simple fanfare
+	[2369] = {effect = CONST_ME_SOUND_YELLOW, item = {id = 2681, count = 10}, chance = 80, remove = true}, -- cornucopia
+	[2370] = {effect = CONST_ME_SOUND_GREEN}, -- lute
+	[2371] = {effect = CONST_ME_SOUND_BLUE}, -- the horn of sundering (actual effect is unknown)
+	[2372] = {effect = CONST_ME_SOUND_GREEN}, -- lyre
+	[2373] = {effect = CONST_ME_SOUND_GREEN}, -- panpipes
+	[3951] = {effect = CONST_ME_SOUND_BLUE}, -- bongo drum (actual effect is unknown)
+	[3952] = {effects = {failure = CONST_ME_POFF, success = CONST_ME_SOUND_GREEN}, chance = 20}, -- didgeridoo
+	[3953] = {effect = CONST_ME_SOUND_RED}, -- war drum
+	[3957] = {effect = CONST_ME_SOUND_YELLOW, item = {id = 2681, count = 10}, chance = 80, remove = true}, -- cornucopia
+	[5786] = {effects = {failure = CONST_ME_SOUND_RED, success = CONST_ME_SOUND_YELLOW}, monster = "war wolf", chance = 60, remove = true}, -- wooden whistle
+	[6572] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
+	[6573] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
+	[13759] = {effect = CONST_ME_SOUND_BLUE} -- small whistle (actual effect is unknown)
+}
+
+local effect, chance = nil, nil
+
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
-	-- TODO: Different music effect for different instruments.
-	item:getPosition():sendMagicEffect(CONST_ME_SOUND_BLUE)
+	local instrument = instruments[item:getId()]
```

Comment:
No, I don't think. As far as you don't add the wrong itemid in actions.xml, which would still be better anyway as it would only help you debugging.